Makefile

```diff
@@ -3,11 +3,9 @@
 check_dirs := optimum test bench examples
 
 check:
-	black --check ${check_dirs}
 	ruff check ${check_dirs}
```

Comment:
AFAIK, the ruff linter only takes care of very specific formatting rules (like sorting imports).
This is why you also need to call `ruff format`.
See an example in another project where I only use ruff" https://github.com/huggingface/optimum-pipelines/blob/main/Makefile.
Also, as you can see in this project, when calling check I added some verbosity options.

Reply:
Thanks @dacorvo for the inputs.
Will call `format` as well.

Also, I think the reference repo that you mentioned (for `check` options) might be private? As I'm getting a code 404 on hitting the link.

Reply:
> Also, I think the reference repo that you mentioned (for check options) might be private? 

Ah yes, sorry. I edited my message to copy of the relevant portion of the Makefile.

Reply:
Thanks!

>Also, as you can see, when calling check I added some verbosity options.

Yeah, makes sense to list out the changes before applying them.